tools/rna_tools/blockclust/test-data/test.tabular

```diff
@@ -0,0 +1,2111 @@
+>cluster_939	chr1	173833959	173834043	-	451.50	51	3
```

Comment:
@pavanvidem this looks wired? is there a line break missing behind the "fasta" header?